jupyterlab/README.md

```diff
@@ -1,14 +1,33 @@
-### Docker build
+### `pip` based version
+
+#### Docker build
+
+```aidl
+docker build -t activeeon/jupyterlab:pip -f ./Dockerfile.pip .
 ```
-docker build -t activeeon/jupyterlab .
+
+#### Docker run
+
+```aidl
+docker run -it -p 8888:8888 activeeon/jupyterlab:pip
```

Comment:
add --name

Reply:
@mboussaa I think it's not necessary into this point...